common/run_as_userid.md

```diff
@@ -0,0 +1,106 @@
+---
+layout: default
+parent: Common
+---
+
+# Run as Userid
+
+Run pyRACF commands as another user.
+{: .fs-6 .fw-300 }
+
+&nbsp;
+
+{: .note}
+> _Changes made using the functionality described here are scoped to the target "Admin" object instance._
+
+&nbsp;
+
+{: .warning}
+> _In order to use `run_as_userid`, the caller must have `UPDATE` access to the `userid.IRRSMO00` resource in the `SURROGAT` class, where `userid` represents the specific userid you wish to execute commands as. Further information can be found outlined in [our dependencies note](../../index)._

```

Comment:
Should this be
```markdown
must have at least `UPDATE` access
```
because if I recall correctly `CONTROL` gives more authority than `UPDATE`, which would also work?

Reply:
ALTER both gives "more" permissions than UPDATE, CONTROL is kinda functionally equivalent to UPDATE depending on what the resource is (CONTROL is either the same, or a "higher authority" in some cases, but no less).

Reply:
So in this case, would all three be sufficient? If so, I think it would probably be a good idea say that the user must have one of those three access levels.

Reply:
Yes, all 3 would be sufficient, but this can be succinctly described as `at least `UPDATE`` which is how it is now reflected here.